### DISCUSSION TITLE
Can I switch workspace between prev & rear?

### DISCUSSION THREAD

**Discussion by HyperCherry** [i3/i3#5409] [Q&A] [answered]
I have used [bspwm](https://github.com/baskerville/bspwm)  before.
Is there any way to switch workspace between previous workspace & rear workspace in i3wm?
Just like `mod + [` and `mod + ]` in bspwm.

**Comment by okraits** [comment, marked as answer]
What is the rear window?

**Reply by HyperCherry** [reply]
Oh,sorry,I means workspace.For example, currently I am in the 3rd workspace. I want to quickly change the current workspace without using `mod + number`. So that I can quickly switch to the 2nd workspace(previous) or the 4th workspace (rear). This is because remembering workspace numbers can be annoying and I want to make it easier.

**Reply by okraits** [reply]
Yes, all available commands are listed in the userguide. Please have a look at it. You can use `workspace next` and `workspace prev` for your usecase:

https://i3wm.org/docs/userguide.html#_changing_named_workspaces_moving_to_workspaces
